hack/verify_boilerplate.py

```diff
@@ -199,7 +199,7 @@ def get_regexs():
     # Search for "YEAR" which exists in the boilerplate, but shouldn't in the real thing
     regexs["year"] = re.compile('YEAR')
     # dates can be 2014, 2015, 2016 or 2017, company holder names can be anything
-    regexs["date"] = re.compile('(2014|2015|2016|2017|2018)')
+    regexs["date"] = re.compile('(2014|2015|2016|2017|2018|2019)')
```

Comment:
Seems like we could do something like `re.compile(r'(%s)' % '|'.join(str(x) for x in range(2014, time.now().year()))` and not have to bother with this in 2020

Reply:
Pls have a look at#10689 where I try to use the script from k/k